data/repositories/identity/src/main/java/com/sanaa/identity/dataSoruce/local/dataStore/PreferencesManager.kt

```diff
@@ -5,8 +5,9 @@ import kotlinx.coroutines.flow.Flow
 interface PreferencesManager {
     val sessionId: Flow<String>
     val isGuest: Flow<Boolean>
+    val accountId: Flow<Long>
     suspend fun updateSessionId(value: String)
     suspend fun setIsGuest(value: Boolean)
-
+    suspend fun setAccountId(value: Long)
```

Comment:
Since the method `updateSessionId(..)` uses the term _update_ it might be more consistent and clearer to rename `setAccountId(..)` to `updateAccountId(..)` 
```suggestion
    suspend fun updateAccountId(value: Long)
```

Reply:
accountId is only set once during login and doesn't change unless the user logs out, so  i think setAccountId better reflects its usage